dataPrepII_PCC/makePCCRateFile.py

```diff
@@ -1,42 +1,34 @@
-import ROOT as r
+import ROOT
 from inputDataReader import *
 import sys, json, pickle
 
 
-def ClusterCount(data):
+def GetCount(data,DataType):
 
-    """This is the method to be called on an event stored by the
-    PixelLumiTupler to compute the number of pixel clusters to be used
-    for the lumi analysis.
-    """
+    if DataType=="VTX":
+        count=data.nVtx
+    else:
+        count=data.nCluster
 
-    # Excludes the innermost barrel layer.
-#    num_clus = data.nPixelClustersB[1] + \
-#               data.nPixelClustersB[2] + \
-#               data.nPixelClustersF[0] + \
-#               data.nPixelClustersF[1]
-
-    num_clus = data.nCluster
-
-    return num_clus
+    return count
```

Comment:
Here, what about an additional argument, e.g. PixelLayer, to give also the option for rates per layer ?

So smth like,
 if PixelLayer>0:
  count = data.nClusterPerLayer[PixelLayer-1]

At least this is what I am doing to make PCC rates per layer ;D